.github/workflows/ci.yml

```diff
@@ -1,4 +1,11 @@
-on: pull_request
+name: CI
```

Comment:
If you do not set the name, the file path will be displayed and it will be difficult to read.

Reply:
Agree. I like the name of "test" you used in your repo slightly better here though!

Reply:
I don't have a strong opinion, but as I mentioned in https://github.com/async-rs/async-std/pull/283#discussion_r331788937, I added all the checks to this workflow, so I think "CI" is fine.